### DISCUSSION TITLE
Does it support a book in the form of sequentially named images in a directory?

### DISCUSSION THREAD

**Discussion by HubKing** [gotson/komga#1112] [Q&A] [answered]
I mean, instead of a file `Test.cbz`, can it recognise a directory named `Test` that contains images like `Test1.jpg`, `Test2.jpg`, `Test3.jpg`,.... as a single book named `Test`? 

I have tried Komga, and it seems that it can only recognise `Test.cbz`, not a directory `Test`.

**Comment by gotson** [comment, marked as answer]
No, it doesn't, and it's not planned.
